### PR TITLE
[Show] Mark non-displayable shows as 404.

### PR DESCRIPTION
**Pull request by alloy**
This will make those errors filtered out and not sent to NewRelic.

### CODE REVIEW

**Comment by joeyAghion** [comment]
Thanks! I think this will help with alert noise.

File: schema/partner_show.js [hunk 1]
```diff
@@ -252,7 +252,11 @@ const PartnerShow = {
   },
   resolve: (root, { id }) => {
     return gravity(`show/${id}`).then(show => {
-      if (!show.displayable) return new Error("Show Not Found")
+      if (!show.displayable) {
+        const error = new Error("Show Not Found")
```
Comment (cavvia) [inline schema/partner_show.js]: Just a general comment - would be nice to use something more specific than `Error`. Maybe we do this all over MP but it's not great practice.

**Comment by cavvia** [comment]
Thanks for fixing this! You might want to rebase on your side as I'm getting a merge warning now.

**Comment by cavvia** [comment]
:+1:

File: schema/partner_show.js (hunk 1 above)
Comment (alloy) [reply schema/partner_show.js]: I agree, in principle. Last night I looked into that, but quickly realised that, as is so often the case, the JS community has many opinions on the matter, not simplified by the prototypical vs class based inheritance debates. Many of them revert to duck-typing.

I took another stab at it and have pushed a working version, but I decided to still check for the existence of the property to identify the error in our error handler and not e.g. `instanceof`.

**Pull request merged**